### PR TITLE
LPS-174758 LPS-202731 Sync with AC API changes

### PR DESCRIPTION
**Pull request by gustavolimav**
https://liferay.atlassian.net/browse/LPS-202731

### CODE REVIEW

**Comment by gustavolimav** [comment]
ci:test:sf

**Comment by gustavolimav** [comment]
ci:test:sf

**Comment by gustavolimav** [comment]
ci:test:search

File: modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/suggestions/spi/asah/BaseAsahSuggestionsContributor.java [hunk 1]
```diff
@@ -190,7 +190,7 @@ protected SuggestionsContributorResults getSuggestionsContributorResults(
 				searchContext, sort, suggestionsContributorConfiguration),
 			"JSONObject/_embedded", "JSONArray/" + path);
 
-		if (jsonArray.length() == 0) {
+		if ((jsonArray == null) || (jsonArray.length() == 0)) {
```
Comment (peerkar) [inline modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/suggestions/spi/asah/BaseAsahSuggestionsContributor.java:193]: Please replace with JsonUtil.isEmpty(JSONArray)

**Review by peerkar — CHANGES_REQUESTED** [review]
@gustavolimav just one small issue there. Otherwise, looks good and ready to go to me.

No further review necessary from my side.

File: modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/suggestions/spi/asah/BaseAsahSuggestionsContributor.java (hunk 1 above)
Comment (gustavolimav) [reply modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/suggestions/spi/asah/BaseAsahSuggestionsContributor.java:193]: Nice It is really better

**Comment by gustavolimav** [comment]
ci:test:sf

**Comment by gustavolimav** [comment]
ci:test:relevant

**Comment by gustavolimav** [comment]
ci:test:search

**Comment by gustavolimav** [comment]
ci:test:stable

**Comment by gustavolimav** [comment]
ci:forward:force

**Pull request closed without merging**